docs/scripts/api/buildComponentsDocumentation.ts

```diff
@@ -203,9 +203,12 @@ function findXDemos(
   return pagesMarkdown
     .filter((page) => page.components.includes(componentName))
     .map((page) => {
+      // Remove link if it exists as the demo title will already be rendered inside a link.
```

Comment:
```suggestion
      // Titles for paid packages have the structure "Page title <a><span/></a>"
      // Next line remove the <a/> to avoid nested links and keep the <span/> to show the plan level.
```